### ISSUE TITLE
World Text Works Inconsistently

### ISSUE THREAD

**Issue by anthonysharpy** [sboxgame/issues#3225]
### Describe the bug

hiya. i seem to recall some updates being done to world text a few weeks ago. it looks much better now but the issue is ever since then it's sort of been doing this thing where it's popping in and out of existence. it seems it sort of shrinks itself down and resets the rotation at certain angles.

https://www.youtube.com/watch?v=llCdOeZzqeo

### To Reproduce

can try this.

i have a feeling the key to reproducing this is using multiple lines of text since single line world text seems to work just fine (although haven't tested it that much).

![image](https://user-images.githubusercontent.com/55365296/235374958-9a771c96-94c7-4446-8825-226d3268d07e.png)

### Expected behavior

doesn't disappear

**Comment by robotboy655** [comment]
I cannot reproduce this by just having some text in a map.

Are you able to create a minimal reproducible setup for this? Or link/send me the map so I can see it happen on my end?
Have you tried loading the map in Sandbox gamemode and seeing if that solves it?

**Comment by anthonysharpy** [comment]
hiya, thanks for looking into this.

it looks like this bug is triggered by code. reproduction steps below:

- add BrokenTerry.txt to the sandbox gamemode (e.g. at facepunch.sandbox/code/entities). rename to BrokenTerry.cs.
- take the map file (rename it to world_text_reproduction.vamp [sorry it won't let me upload these file types on github]). build the map, no lighting or vis needed probably.
- run the map in sandbox.
- should get the below.

![image](https://user-images.githubusercontent.com/55365296/236342792-37d4b352-3bc5-4622-a153-be411e07409b.png)

basically, creating a WorldPanel through code with a map that has world text in it seems to break the world text. not sure if creating the WorldPanel via an entity is important, you might be able to reproduce it some other way.

[world_text_reproduction.txt](https://github.com/sboxgame/issues/files/11402035/world_text_reproduction.txt)
[BrokenTerry.txt](https://github.com/sboxgame/issues/files/11402028/BrokenTerry.txt)

**Comment by robotboy655** [comment]
Thanks, it should be fixed when the game next updates. Your game(s) will need to be reuploaded to asset.party.

For future, you can upload `.zip` files to GitHub I think.
